scripts/build/build/__init__.py

```diff
@@ -25,7 +25,8 @@ def __init__(self, runner, repository_path: str, output_prefix: str):
         self.completed_steps = set()
 
     def SetupBuilders(self, targets: Sequence[str],
-                      enable_flashbundle: bool):
+                      enable_flashbundle: bool,
```

Comment:
This stars going down a path of adding more and more arguments. Lets remove that path from the start: please add a 'BuilderOptions' dataclass that has enable_flashbundle and pw_command_launcher and pass that in here and inside choice::Create. That way we do not have so many arguments and if we pass more options only the object changes and not the rest of the things. choice::Create already started jumping from 5 arguments to 6.

Reply:
Done